src/examples/iris/helper.cpp

```diff
@@ -26,10 +26,10 @@ void readIrisData(const std::string fileName,
   }
   std::string line;
   std::string value;
-  while(marian::io::getline(in, line)) {
+  while(std::getline(in, line)) {
     std::stringstream ss(line);
     int i = 0;
-    while(marian::io::getline(ss, value, ',')) {
+    while(std::getline(ss, value, ',')) {
```

Comment:
This should be done rather with split?

Reply:
Why not just leaving `std::getline` here?

Reply:
It's not a line. Seems wrong semantically.

Reply:
I think it's fine using generalized getline to scan for a character.  The real evil is std::stringstream when one should be using a `string_view`.